src/components/Header/Header.jsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from "react";
 import { Link } from 'react-router-dom';
-import styles from './Header.module.css';  
+import icon_close from "../../assets/icon-close.svg";
+import logo from "../../assets/logo.svg";
+import styles from "./Header.module.css";
 
 const Header = () => {
+  const [mobileMenu, setMobileMenu] = useState(false);
+  const toggleMenu = () => {
+    mobileMenu ? setMobileMenu(false) :
+   setMobileMenu(true);
+  };
   return (
+  
     <header className={styles.header}>
-      <nav>
-        <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-          <li><Link to="/contact">Contact Us</Link></li>
+      <img src={logo} alt="myteam logo" className={styles.logo} />
+      <nav className={styles.header-nav}>
+        <ul className={ mobileMenu ? '' :  styles['hide-mobile-menu']}>
```

Comment:
This `ul` needs to be `display: flex;` and some other stuff, so we need to have the styles you already added as `.nav-list` get applied here _all the time_. So we need to conditionally add multiple classes. The classNames library is already doing a similar thing in the Button utility component; you can take a look [here](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/blob/6f78123f16138af0b45ce0bc4fffb3e7a1270b7a/src/components/utility/Button/Button.jsx#L22-L24).

Add `import classNames from "classnames";` in the imports at the top of the file

then after the mobile menu toggle and before the JSX, add this:
```
  const listClass = classNames(styles['nav-list'], {
    [styles['hide-mobile-menu']]: mobileMenu,
  });
```